src/main/webapp/script.js

```diff
@@ -0,0 +1,31 @@
+function loadBody() {
+  console.log("Verifying login...");
```

Comment:
This is probably okay since it's prototype code but in the future you won't want to console log stuff. Maybe good to leave a comment here reminding you to remove this later.

Reply:
Ditto below